batcher/aligned-batcher/.env.dev

```diff
@@ -2,7 +2,7 @@ AWS_SECRET_ACCESS_KEY=test
 AWS_REGION=us-east-2
 AWS_ACCESS_KEY_ID=test
 AWS_BUCKET_NAME=aligned.storage
-STORAGE_ENDPOINT=http://localhost:4566/aligned.storage
+STORAGE_ENDPOINT="https://mina-bridge.s3.us-west-2.amazonaws.com"
```

Comment:
I think we should leave this var unchanged and set it to the value we want in `.env`

Reply:
I agree with leaving this unchanged!